### DISCUSSION TITLE
Using Photomaker V1.0 but with V2.0 setup

### DISCUSSION THREAD

**Discussion by gedasv** [TencentARC/PhotoMaker#185] [General]
Hi! Is it possible to setup the V2.0 with the V1.0 training? Also remove insightface library completely? If it's possible to setup, how to do that?

The main motivation is to use the new V2.0 while avoiding any license problems.

**Comment by jm-01az** [comment]
![IMG-20250113-WA0009](https://github.com/user-attachments/assets/a3162fdc-1272-4bad-a37f-c54aa43c3c1a)
a youn man with elegant social cothe ,with a gray suit
